app/controllers/hyrax/file_sets_controller.rb

```diff
@@ -162,11 +181,13 @@ def attributes
 
     def presenter
       @presenter ||= begin
-        _, document_list = search_results(params)
-        curation_concern = document_list.first
-        raise CanCan::AccessDenied unless curation_concern
-        show_presenter.new(curation_concern, current_ability, request)
-      end
+                       _, document_list = search_results(params)
```

Comment:
There's some echoes of commonality between this and the above comment.  Not a blocker, but one I'm noodling on.